documents/src/pages/elements/search-field.md

```diff
@@ -240,3 +252,37 @@ searchInput.addEventListener("value-changed", (e) => {
     debounce(search(e.detail.value), 1500) // debounce search() for 1.5 seconds
 });
 ```
+
+## Accessibility
+::a11y-intro::
+
+`ef-search-field` is assigned `role="textbox"`. States such as `disabled` or `read-only` are programmatically updated to match the element’s visual state. Dynamic updates such as a validation message are communicated to screen readers through a live region. 
```

Comment:
```suggestion
`ef-search-field` is assigned `role="textbox"`. States such as `disabled` or `readonly` are programmatically updated to match the element’s visual state. Dynamic updates such as a validation message are communicated to screen readers through a live region. 
```

Reply:
I think it is also worth mentioning that when `icon-has-action` the magnifier icon as focusable.

Reply:
Don't feel that developers need to know it, do they?